denylist/src/client.rs

```diff
@@ -0,0 +1,66 @@
+use crate::Result;
+use crate::{error::Error, models::metadata::DenyListMetaData};
+use std::str;
+use std::time::Duration;
+
+/// The default client useragent for denylist http requests
+static DEFAULT_USERAGENT: &str = "https://github.com/helium/miner";
```

Comment:
is this actually being verified against anything? we should probably make it something more specific to the repo instead of referencing the old miner one (which doesn't technically meet the standards for user-agent anyway)

Reply:
No it's not verified.. it's just useful when you ask GH questions/reports about origins.. make it something more user agent centric and related to this oracle